addons/payment/migrations/12.0.1.0/post- 😎.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def map_payment_transaction_state(cr):
+    openupgrade.map_values(
+        cr,
+        openupgrade.get_legacy_name('state'),
+        'state',
+        [('refunding', 'pending'),
+         ('refunded', 'done'),
```

Comment:
It should be `'cancel'`, not `'done'`:

v11:
https://github.com/OCA/OpenUpgrade/blob/65767162fcd60037a79532b31df0c5f253b6f74d/addons/payment_authorize/models/payment.py#L336
v12:
https://github.com/OCA/OpenUpgrade/blob/e2fdf7e64894efc786e4b56df2899087a0925398/addons/payment_authorize/models/payment.py#L320

https://github.com/OCA/OpenUpgrade/blob/e2fdf7e64894efc786e4b56df2899087a0925398/addons/payment/models/payment_acquirer.py#L762

The same may apply to `refunding`. At the end, it means cancel.

Reply:
I am not sure, but if you say so...

Reply:
Yes, I have spent 10 minutes following the trace of this.

Reply:
Good 👍😂